test/statistics/StatisticsTest.js

```diff
@@ -227,3 +234,21 @@ function getClientStatisticsFromServer(cluster, client) {
         return null;
     });
 }
+
+function extractString(stats, statName) {
```

Comment:
As far as I understand this method extracts the value of given statsName. If that is so I think renaming it as `extractStatValue` would be better.